<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#QWidget{
	background-color: rgb(255, 255, 255);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>528</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>528</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>528</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>1071</width>
       <height>541</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>528</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame{border-image: url(:/new/home/home/bj.png);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="bt_liulan">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>260</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>浏览数据</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/home/home/浏览数据.png</normaloff>:/new/home/home/浏览数据.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>460</y>
        <width>1051</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_2{background-color: rgb(15, 100, 165);}</string>
      </property>
      <widget class="QLabel" name="l">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>检测医师</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>检测地址</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_doc">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>35</y>
         <width>143</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 211, 243);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_addr">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>36</y>
         <width>251</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 211, 243);</string>
       </property>
      </widget>
     </widget>
     <widget class="QToolButton" name="bt_kuaisu">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>快速检测</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/home/home/快速检测.png</normaloff>:/new/home/home/快速检测.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="bt_fuhua">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>孵化检测</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/home/home/孵化检测.png</normaloff>:/new/home/home/孵化检测.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <widget class="widget_progress" name="wg_progress" native="true">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>600</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>510</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>414</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>孵化时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_time">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>410</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="bt_quxiao">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="MainButton" name="bt_help">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 帮助</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/home/home/帮助.png</normaloff>:/new/home/home/帮助.png</iconset>
    </property>
   </widget>
   <widget class="MainButton" name="bt_saomiao">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 扫描检测</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/home/home/扫描检测.png</normaloff>:/new/home/home/扫描检测.png</iconset>
    </property>
   </widget>
   <widget class="MainButton" name="bt_canshu">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>570</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 参数</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/home/home/参数.png</normaloff>:/new/home/home/参数.png</iconset>
    </property>
   </widget>
   <widget class="MainButton" name="bt_shutup">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 关机</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/home/home/关机.png</normaloff>:/new/home/home/关机.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>widget_progress</class>
   <extends>QWidget</extends>
   <header>widget_progress.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
